packages/common-db/src/lib/common/session.ts

```diff
@@ -59,21 +66,22 @@ export function createAdapterStore(session: Session, defaultTtl = 3600): any {
          * @param sess the session
          * @param fn callback
          */
-        set(sid: string, ttl: number, sess: Session, fn: (args: any) => void): void {
+        set(
```

Comment:
you should define the exact parameter options that can happen and then just have this one as implementation. so define all sets that are allowed which should be just 2 different right?

set(sid, ttl, sess, fn) and
set(sid, sess, fn)

without that the typing would also sallow to call 
set(sid, ttl, fn, fn)

which might not be intended or?